api/docs/v2/storefront/index.yaml

```diff
@@ -307,32 +382,71 @@ components:
               properties:
                 number:
                   type: string
+                  example: 'R123456789'
+                email:
+                  type: string
+                  example: 'spree@example.com'
                 item_total:
                   type: string
+                  example: '19.99'
+                display_item_total:
+                  type: string
+                  example: '$19.99'
                 total:
                   type: string
+                  example: '29.99'
+                display_total:
+                  type: string
+                  example: '$29.99'
                 ship_total:
                   type: string
+                  example: '0.00'
+                display_ship_total:
+                  type: string
+                  example: '$19.99'
                 adjustment_total:
                   type: string
-                created_at:
+                  example: '10.00'
+                display_adjustment_total:
                   type: string
+                  example: '$10.00'
+                created_at:
+                  $ref: '#/components/schemas/Timestamp'
                 updated_at:
-                  type: string
+                  $ref: '#/components/schemas/Timestamp'
                 included_tax_total:
                   type: string
+                  example: '5.00'
                 additional_tax_total:
                   type: string
+                  example: '5.00'
                 display_additional_tax_total:
                   type: string
+                  example: '$5.00'
                 display_included_tax_total:
                   type: string
+                  example: '$5.00'
                 tax_total:
                   type: string
+                  example: '10.00'
+                display_tax_total:
+                  type: string
+                  example: '$10.00'
+                total_quantity:
+                  type: number
+                  example: 2
+                  description: 'Total quantity number of all items added to the Cart'
+                special_instructions:
+                  type: string
+                  example: 'Please ship ASAP!!!111'
```

Comment:
Can we provide more solid example here? 😄

Reply:
@kushniryb why so serious? ;)